test/classes/Controllers/Sql/SetValuesControllerTest.php

```diff
@@ -39,21 +39,23 @@ public function testError(): void
     {
         $this->dummyDbi->addResult('SHOW COLUMNS FROM `cvv`.`enums` LIKE \'set\'', false);
 
-        $_POST = [
-            'ajax_request' => true,
-            'db' => 'cvv',
-            'table' => 'enums',
-            'column' => 'set',
-            'curr_value' => 'b&c',
-        ];
-        $GLOBALS['db'] = $_POST['db'];
-        $GLOBALS['table'] = $_POST['table'];
+        $request = $this->createStub(ServerRequest::class);
+        $request->method('getParsedBodyParam')->willReturnMap([
+            ['ajax_request', null, true],
+            ['db', null, 'cvv'],
+            ['table', null, 'enums'],
+            ['column', null, 'set'],
+            ['curr_value', null, 'b&c'],
+        ]);
+
+        $GLOBALS['db'] = $request->getParsedBodyParam('db');
+        $GLOBALS['table'] = $request->getParsedBodyParam('table');
```

Comment:
You can use the literal string here.
```suggestion
        $GLOBALS['db'] = 'cvv';
        $GLOBALS['table'] = 'enums';
```